### ISSUE TITLE
[20 BETA] keyboard navigation in file browser doesn't work if there is only one file in it

### ISSUE THREAD

**Issue by atar-axis** [linuxdeepin/developer-center#1730]
## Describe the bug
While navigating through your files, using the keyboard:
As soon as you reach a folder with only one other folder in it, you cannot use the starting letter anymore to select the folder.

## To Reproduce
see screencast

## Screenshots
![Screen Capture_dde-file-manager_20200514152136](https://user-images.githubusercontent.com/11148588/81939902-3bbe2800-95f7-11ea-8111-ae608bab3de9.gif)

### Distro & Version
Linux Deepin 20 beta

**Comment by Zeno-sole** [comment]
@babyfengfjx  该bug确实存在，请与文管团队确认后提单

**Comment by SuperEffie** [comment]
Thank you for your feedback, the problem has been confirmed to exist in the Community20.4 version,  it has been reported to the developer, and you will be notified here after the fix

**Comment by SuperEffie** [comment]
At present, the deepin has been upgraded to version 20.8. It is confirmed that this problem has been fixed.  I close this issue at this moment. Welcome to feedback more issue for us.Thank you~
Best wishes!
